2021-12-01 15:47:32,686 INFO    MainThread:1670489 [wandb_setup.py:_flush():69] setting env: {}
2021-12-01 15:47:32,686 INFO    MainThread:1670489 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-01 15:47:32,687 INFO    MainThread:1670489 [wandb_init.py:_log_setup():357] Logging user logs to /home/lts5-ts/Documents/ahmed/bmai_ahmed/wandb/run-20211201_154732-tpbrtzn5/logs/debug.log
2021-12-01 15:47:32,687 INFO    MainThread:1670489 [wandb_init.py:_log_setup():358] Logging internal logs to /home/lts5-ts/Documents/ahmed/bmai_ahmed/wandb/run-20211201_154732-tpbrtzn5/logs/debug-internal.log
2021-12-01 15:47:32,687 INFO    MainThread:1670489 [wandb_init.py:init():390] calling init triggers
2021-12-01 15:47:32,687 INFO    MainThread:1670489 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-12-01 15:47:32,687 INFO    MainThread:1670489 [wandb_init.py:init():435] starting backend
2021-12-01 15:47:32,687 INFO    MainThread:1670489 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-01 15:47:32,704 INFO    MainThread:1670489 [backend.py:ensure_launched():135] starting backend process...
2021-12-01 15:47:32,710 INFO    MainThread:1670489 [backend.py:ensure_launched():139] started backend process with pid: 1670497
2021-12-01 15:47:32,711 INFO    MainThread:1670489 [wandb_init.py:init():440] backend started and connected
2021-12-01 15:47:32,711 INFO    MainThread:1670489 [wandb_init.py:init():499] updated telemetry
2021-12-01 15:47:32,712 INFO    MainThread:1670489 [wandb_init.py:init():522] communicating current version
2021-12-01 15:47:33,140 INFO    MainThread:1670489 [wandb_init.py:init():527] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-01 15:47:33,141 INFO    MainThread:1670489 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-12-01 15:47:33,334 INFO    MainThread:1670489 [wandb_init.py:init():562] starting run threads in backend
2021-12-01 15:47:34,600 INFO    MainThread:1670489 [wandb_run.py:_console_start():1651] atexit reg
2021-12-01 15:47:34,601 INFO    MainThread:1670489 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.REDIRECT
2021-12-01 15:47:34,601 INFO    MainThread:1670489 [wandb_run.py:_redirect():1530] Redirecting console.
2021-12-01 15:47:34,602 INFO    MainThread:1670489 [wandb_run.py:_redirect():1586] Redirects installed.
2021-12-01 15:47:34,602 INFO    MainThread:1670489 [wandb_init.py:init():587] run started, returning control to user process
2021-12-01 15:47:36,148 INFO    MainThread:1670489 [wandb_watch.py:watch():43] Watching
2021-12-01 15:47:36,150 INFO    MainThread:1670489 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 1
2021-12-01 15:47:36,150 INFO    MainThread:1670489 [wandb_run.py:_restore():1593] restore
2021-12-01 15:47:38,630 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 980
}

2021-12-01 15:47:38,899 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 980
}

2021-12-01 15:47:39,365 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 10571
}

2021-12-01 15:47:39,468 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 10571
}

2021-12-01 15:47:39,569 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 10571
}

2021-12-01 15:47:39,672 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-12-01 15:47:39,775 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-12-01 15:47:39,880 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-12-01 15:47:39,983 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-12-01 15:47:40,086 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-12-01 15:47:40,434 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}

2021-12-01 15:47:40,863 INFO    MainThread:1670489 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10571
  total_bytes: 10571
}
local_info {
}

2021-12-01 15:47:41,941 INFO    MainThread:1670489 [wandb_run.py:_append_files():2009] logging synced files
